pkg/spec/postgresql.go

```diff
@@ -96,9 +96,11 @@ type PostgresSpec struct {
 	TeamID              string   `json:"teamId"`
 	AllowedSourceRanges []string `json:"allowedSourceRanges"`
 	DockerImage         string   `json:"dockerImage,omitempty"`
-	// EnableLoadBalancer  is a pointer, since it is important to know if that parameters is omitted from the manifest
-	UseLoadBalancer     *bool                `json:"useLoadBalancer,omitempty"`
-	ReplicaLoadBalancer bool                 `json:"replicaLoadBalancer,omitempty"`
+	// EnableLoadBalancer  is a pointer, since it is important to know if that parameters is omitted from the Postgres manifest
+	// in that case UseLoadBalancer == nil and the value is taken from the operator config
+	UseLoadBalancer *bool `json:"useLoadBalancer,omitempty"`
+	// if ReplicaLoadBalancer == nil (is unset), value of UseLoadBalancer determines if a balancer for replicas is created
+	ReplicaLoadBalancer *bool                `json:"replicaLoadBalancer,omitempty"`
```

Comment:
Just a rhetorical question - why so unobvious, true/false and a nil pointer? Maybe it would be more straightforward to introduce some sort of enum True/False/Unset ?

Reply:
Why do we need to know where the value is coming from?